protos/mission/mission.proto

```diff
@@ -182,6 +183,16 @@ message MissionItem {
         CAMERA_ACTION_START_VIDEO = 4; // Start capturing video
         CAMERA_ACTION_STOP_VIDEO = 5; // Stop capturing video
     }
+
+    enum PlaneAction {
+        VEHICLE_ACTION_NONE = 0;
+        VEHICLE_ACTION_ARM = 1;
+        VEHICLE_ACTION_DISARM = 2;
+        VEHICLE_ACTION_TAKEOFF = 3;
+        VEHICLE_ACTION_LAND = 4;
```

Comment:
Are these usable by multicopter as well? If so, this should be called VehicleAction (everwhere) and not PlaneAction.

Reply:
I agree, I forgot to rename it, in latest commit ist is already named VehicleAction.